kitchen/ph5tostationxml.py

```diff
@@ -401,7 +401,7 @@ def create_obs_channel(self, station_list, deployment, cha_code, loc_code,
         if self.ph5.Das_t.has_key(das):
             Das_t = ph5API.filter_das_t(self.ph5.Das_t[das]['rows'],
                                         station_list[deployment][0]['channel_number_i'])
-            Receiver_t=self.ph5.get_receiver_t (Das_t[0], by_n_i=True)
+            Receiver_t=self.ph5.get_receiver_t (Das_t, by_n_i=True)

```

Comment:
hmm that's very strange. I get that same error on my machine when the [0] isn't there. let's change it back for now since the [0] works fine in web services and I'll figure out what is goig on